templates/zerver/integrations/github-actions.md

```diff
@@ -1,8 +1,60 @@
-It's easy to send Zulip messages from GitHub Actions! This is useful:
+# Zulip GitHub Actions integration
 
-  * If you want to report the success or failure of an Action to Zulip.
-  * If you want to send output from your actions to Zulip.
+Get Zulip notifications from GitHub Actions!
 
-Instructions are available [here](https://github.com/zulip/github-actions-zulip#readme).
+{start_tabs}
 
-See also the [GitHub integration](/integrations/doc/github).
+1.  {!create-a-generic-bot.md!}
+
+1.  To send Zulip notifications whenever your workflow runs, add the
+    `zulip/github-actions-zulip/send-message@v1` action to your GitHub
+    Actions workflow file, and set the input values as specified in the
```

Comment:
I wonder about "as specified" here. The message content for the template is for a specific case. Might be better to be something more generic? Or note that the content would be for a specific case?

I assume that when setting these up, you would actually set these notifications to send whenever a particular thing happens when the workflow runs. The start of this instruction makes it sound like you're setting something to send every time the action runs.